src/Workspaces/CSharpTest/CodeGeneration/AddImportsTests.cs

```diff
@@ -1600,102 +1602,102 @@ class C
 }", safe: true, useSymbolAnnotations);
         }
 
-        [Theory, InlineData(true), InlineData(false)]
-        public async Task TestWarnsWithMatchingExtensionMethodUsedAsDelegate(bool useSymbolAnnotations)
-        {
-            var source = @"using System;
-using B;
-
-namespace A
-{
-    static class AExtensions
-    {
-        public static void M(this int a){}
-    }
-    public class C1 {}
-}
-
-namespace B
-{
-    static class BExtensions
-    {
-        public static void M(this object a){}
-    }
-}
-
-class C
-{
-    Action M(A.C1 c1) => 42.M;
-}";
-
-            await TestAsync(
-                source,
-@"using System;
-using A;
-using B;
-
-namespace A
-{
-    static class AExtensions
-    {
-        public static void M(this int a){}
-    }
-    public class C1 {}
-}
-
-namespace B
-{
-    static class BExtensions
-    {
-        public static void M(this object a){}
-    }
-}
-
-class C
-{
-    Action M(A.C1 c1) => 42.M;
-}",
-
-@"using System;
-using A;
-using B;
-
-namespace A
-{
-    static class AExtensions
-    {
-        public static void M(this int a){}
-    }
-    public class C1 {}
-}
-
-namespace B
-{
-    static class BExtensions
-    {
-        public static void M(this object a){}
-    }
-}
-
-class C
-{
-    Action M(C1 c1) => 42.M;
-}", safe: true, useSymbolAnnotations);
-
-            var doc = await GetDocument(source, useSymbolAnnotations);
-            OptionSet options = await doc.GetOptionsAsync();
-
-            var imported = await ImportAdder.AddImportsFromSyntaxesAsync(doc, true, options);
```

Comment:
this test is bogus because of this line.  there is no way in the product to call `AddImportsFromSyntaxesAsync` with `safe: true`.  So this test was not testing a real scenario.